README.md

```diff
@@ -131,14 +140,15 @@ If you can't use the online cloud Vault:
 ## Robot run
 
 Run with **VSCode** or **rcc** the following tasks in order:
-1. `Init Google/Microsoft OAuth`: Opens a browser window for you to authenticate and
+1. `Init Google` / `Microsoft OAuth`: Opens a browser window for you to authenticate and
```

Comment:
This might create some confusion, as there's no individual `Init Google` or `Microsoft OAuth` task.

What about:
```suggestion
1. `Init Google OAuth` / `Init Microsoft OAuth`: Opens a browser window for you to authenticate and
```
?

And make sure you apply the same idiom in other places like [Tasks](https://github.com/robocorp/example-oauth-email/tree/update-readme-per-feedback#tasks) as well:
<img width="877" alt="Screenshot 2023-06-15 at 11 03 27" src="https://github.com/robocorp/example-oauth-email/assets/709053/49fa1da8-fa25-4e92-8e02-a395876374b3">